appf.adoc

```diff
@@ -510,7 +510,7 @@ Corresponds to a OGC WKT PROJCS node name.
 
 | **`reference_ellipsoid_name`** | S
 | The name of the reference ellipsoid.
-Valid names are given in https://github.com/cf-convention/cf-conventions/wiki/Mapping-from-CF-Grid-Mapping-Attributes-to-CRS-WKT-Elements (ellipsoid.csv).
+Valid names are given in <<CF-WKT>> (ellipsoid.csv).
```

Comment:
I cannot find the file `ellipsoid.csv` in `<<CF-WKT>>`. THis one I was able to locate and download from somewhere about a year ago, but I do not remember the location (and I seem not to be able to retrace what I did).

Reply:
Thanks for pointing this out. Please see #537 for resolution.